src/EnergyPlus/DataConvergParams.hh

```diff
@@ -101,6 +101,10 @@ namespace DataConvergParams {
 
 		// Default Constructor
 		HVACNodeConvergLogStruct() :
+			NodeNum( 0 ),
+			NotConvergedHumRate( false ),
```

Comment:
@DeadParrot These three NotConverge* variables never seem to be used.  @EnergyArchmage Can we delete them and the loops that are setting them?  Or did I miss something?